src/utils/data-formatter.js

```diff
@@ -1,15 +1,40 @@
 export function formatProducts(product) {
   return {
     ...product,
-    salePrice: `${(product.salePrice).toFixed(2)}`
+    salePrice: `${product.salePrice.toFixed(2)}`,
   };
 }
 
+export function dateFormatter(date) {
+  const selectedDate = new Date(date);
+  const year = selectedDate.getFullYear();
+  const month = String(selectedDate.getMonth() + 1).padStart(2, "0");
+  const day = String(selectedDate.getDate()).padStart(2, "0");
+
+  return `${day}/${month}/${year}`;
+}
 
 export function userPropsFormatter(user) {
   const name = (user.firstName || "") + " " + (user.lastName || "");
   return {
     title: name,
     subtitle: user.email,
   };
-}
\ No newline at end of file
+}
+
+export function mapUserDataToNewData(userData) {
+  const newUserData = {};
+  Object.keys(userData.value).forEach((key) => {
+    if (key === "birthDate") {
+      newUserData[key] = dateFormatter(userData.value[key]);
+    } else {
+      newUserData[key] = userData.value[key];
+    }
+  });
+  return newUserData;
```

Comment:
Wait, why do we need this? doesn't this return just a copy of `userData` with formatted birthday field?

Reply:
yes, with one of the methods, Im converting the date that comes from the input field to generate it in dd/mm/yyyy so the BE can accept it, in the Update view im reversing it. That way im removing the warning for the dates